src/mudlet.cpp

```diff
@@ -2299,7 +2299,10 @@ void mudlet::slot_update_shortcuts()
             return;
         }
     }
+    assignKeySequences();
+}
 
+void mudlet::assignKeySequences() {
```

Comment:
Just a code style tweak:
```suggestion
void mudlet::assignKeySequences()
{
```